.gitlab-ci.yml

```diff
@@ -1,27 +1,77 @@
 image: solvcon/solvcon_build:latest
```

Comment:
See https://gitlab.com/yungyuc/solvcon/pipelines/26228131 to see the working the pipeline